src/ServiceControl/WebApplicationBuilderExtension.cs

```diff
@@ -132,6 +133,14 @@ public static void AddServiceControl(this WebApplicationBuilder hostBuilder, Set
             }
 
             hostBuilder.AddServiceControlComponents(settings, ServiceControlMainInstance.Components);
+
+            hostBuilder.AddThroughputCollector(broker: TransportManifestLibrary.GetName(settings.TransportType),
+               serviceControlAPI: settings.ApiUrl,
+               errorQueue: settings.ErrorQueue,
+               auditQueue: "?",
+               transportConnectionString: settings.TransportConnectionString,
+               persistenceType: PersistenceManifestLibrary.GetName(settings.PersistenceType)
```

Comment:
Is this even required? We know we are going to use Raven, correct?

Reply:
It is as when the main instance changes to have options, we also need to have the option. Currently I think we also do an InMemory persister for testing, so that would be passed in.

Reply:
>It is as when the main instance changes to have options

There is an assumption here that, as we know, may not be true 😄